PowerShell/JumpCloud Module/Public/Users/Remove-JCUser.ps1

```diff
@@ -26,39 +26,64 @@ UserID has an Alias of _id. This means you can leverage the PowerShell pipeline
             HelpMessage = 'Use the -ByID parameter when the UserID is passed over the pipeline to the Remove-JCUser function. The -ByID SwitchParameter will set the ParameterSet to ''ByID'' which will increase the function speed and performance.')]
         [Switch]
         $ByID,
-
+        # Do not use $CascadeManager if $force is used
         [Parameter(HelpMessage = 'A SwitchParameter which suppresses the warning message when removing a JumpCloud User.')]
         [Switch]
-        $force
+        $force,
+        [Parameter(HelpMessage = 'A SwitchParameter for Cascading the manager of the user to the users managed by the user. NULL, AUTOMATIC (bubble up), ID (prompt for manager ID)')]
+        [ValidateSet('NULL', 'Auto', 'ID')]
```

Comment:
```suggestion
        [ValidateSet('Null', 'Automatic', 'Id')]
```

Let's talk about this as a team but I want these to be same case. Should we keep these param sets as is? Do they make sense for the customer?

Reply:
I'm slightly torn on this.

At first glance when testing I was like this doesn't make much sense but I understand why would do it this way as well. If we want tab completion we want to use ValidateSet, but we could potentially leverage ValidateScript and check for $null, Automatic/Auto and the pattern for a userID and then forego the dynamicparams

Just a thought